### ISSUE TITLE
Consider grouping Cloud-init/Sysprep and Start VM on creation in Advanced Options of the Basic settings

### ISSUE THREAD

**Issue by hstastna** [oVirt/ovirt-web-ui#1297]
Related comment: https://github.com/oVirt/ovirt-web-ui/pull/1279#discussion_r490348578

Consider grouping _Enable Cloud-init/Sysprep_ and _Start virtual machine on creation_ options (enabled by checking appropriate checkboxes) in a new expand/collapse _Advanced Options/Advanced CPU Topology Options_ section of the Create VM Wizard's _Basic settings_ step:
![advanced](https://user-images.githubusercontent.com/13417815/93623695-65d98980-f9df-11ea-9a4d-60104470756e.png)

This suggestion was raised because Cloud-init option is already part of the _Advanced Options_ of the VM _Details_ card, together with the _vCPU Topology_ options:
![cloudinit](https://user-images.githubusercontent.com/13417815/93624707-0b412d00-f9e1-11ea-8a94-316f83f4bfd2.png)

So we could make those options in the Basic settings more consistent with those in the VM Details card. However, with this change, we will not be able to see the _Enable Cloud-init/Sysprep_ and _Start virtual machine on creation_ options by default, only after expanding the Advanced Options section.

**Comment by hstastna** [comment]
@rszwajko ⬆️ I think this is exactly what you've already suggested to me. However, as Scott has mentioned [here](https://github.com/oVirt/ovirt-web-ui/pull/1279#discussion_r490348578), there's also a disadvantage of this approach: we will not see those options all the time and probably we do want to see them.
@sgratch @bamsalem Feel free to add any suggestions, comments.

**Comment by rszwajko** [comment]
How about replacing also checkboxes with toggles (similar as in VM details)? This way the labels are on the left in the same
column as all other labels which makes them more distinct from other form elements.

**Comment by sgratch** [comment]
@hstastna After discussing this on grooming meeting we think that this is not worth the time for fixing and it's ok to leave it as is.
On VMs details card there is a group of advanced options to set (not just cpu arc and syspre/cloud-init) and it was decided that those options are less frequently updated for an existing VM so all those were put together under the advanced section).

On create VM for VM portal, using the cloud-init/sysprep settings for initial run, based on OS,  should be more frequently used or at least represented to the user (and not hidden under the advanced option) and the same goes for auto start. If it will be hidden under the advanced then the user won't easily know about this option. 
The cpu  topology is really an advanced option and it's kept under advanced section on webadmin as well.

Considering all that and till we'll add more settings to this dialog which require free some space, I think it's ok to leave it as is for now.

@rszwajko regarding replacing checkboxes with toggles, I don't have a strong opinion on this as long as we are consistent in replacing that for all other places like bootable.

**Comment by hstastna** [comment]
@sgratch Ok, I understand, thank you for the explanation. We can leave it as it is. Should I close this issue?

**Comment by sgratch** [comment]
Closed as WONTFIX due to [comment](https://github.com/oVirt/ovirt-web-ui/issues/1297#issuecomment-713670901).
